armi/reactor/tests/test_reactors.py

```diff
@@ -334,6 +334,51 @@ def test_findAllAxialMeshPoints(self):
         blockMesh = self.r.core.getFirstAssembly(Flags.FUEL).spatialGrid._bounds[2]
         assert_allclose(blockMesh, mesh)
 
+    def test_findAllAziMeshPoints(self):
+        aziPoints = self.r.core.findAllAziMeshPoints()
+        expectedPoints = [
+            -50.7707392969,
+            -36.2648137835,
+            -21.7588882701,
+            -7.2529627567,
+            7.2529627567,
+            21.7588882701,
+            36.2648137835,
+            50.7707392969,
+            65.2766648103,
+            79.7825903236,
+            94.288515837,
+            108.7944413504,
+            123.3003668638,
+        ]
+        self.assertListEqual(expectedPoints, aziPoints)
```

Comment:
We sometimes use things like `np.testing.assert_almost_equal` for list comparisons of floats, which may twiddle sometimes. 

There's also some ongoing concern about how to handle the maintenance burden of tests like this that change when the test fixtures change. We have lots like this so you're doing as we always do, but it's just something we worry about. We'll probably eventually need to decouple more tests from the "one test reactor" loaded in this case, even though we thought it was convenient at first. Just a fyi.